onnxruntime/core/session/plugin_ep/ep_factory_provider_bridge.cc

```diff
@@ -20,6 +22,11 @@ OrtStatus* ProviderBridgeEpFactory::GetSupportedDevices(EpFactoryInternal& ep_fa
     auto* ep_device = ep_devices[i];
     if (ep_device) {
       ep_device->ep_factory = &ep_factory;
+
+      // Add library path to EP metadata if available
```

Comment:
Who uses this metadata? I only see this key-value pair is set. I don't see how the value is used. And there is no test case or example.